modules/Microsoft.EventGrid/topics/eventSubscriptions/readme.md

```diff
@@ -1,20 +1,20 @@
-# EventGrid EventSubscriptions `[Microsoft.EventGrid/eventSubscriptions]`
+# EventGrid Topics EventSubscriptions `[Microsoft.EventGrid/topics/eventSubscriptions]`
 
-This module deploys EventGrid EventSubscriptions.
+This module deploys EventGrid Topics EventSubscriptions.
+// TODO: Replace Resource and fill in description
```

Comment:
Leftovers. Also please update so the title & description as per the suggestions for the other readme